picard/extension_points/item_actions.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+#
+# Picard, the next-generation MusicBrainz tagger
+#
+# Copyright (C) 2024 Laurent Monin
```

Comment:
If we want to be correct here I think this should be taken from the original file.